contrib/platform/test/com/sun/jna/platform/win32/Kernel32UtilTest.java

```diff
@@ -513,4 +513,38 @@ public void testGetLogicalProcessorInformationEx() {
             assertTrue(cache.associativity == WinNT.CACHE_FULLY_ASSOCIATIVE || cache.associativity > 0);
         }
     }
+
+    public void testSetCurrentProcessPriority() {
+        try {
+            Kernel32Util.setCurrentProcessPriority(Kernel32.HIGH_PRIORITY_CLASS);
+        } catch (Win32Exception e) {
+            fail("setCurrentProcessPriority() failed: " + e.getMessage());
+        }
```

Comment:
I don't think we have this pattern anywhere else in tests - I wouldn't catch and assert an expectation like elsewhere. If there's an exception the test will fail anyway.

```java
assertTrue( Kernel32Util.setCurrentProcessPriority(Kernel32.HIGH_PRIORITY_CLASS));
```